website/content/docs/terminology.mdx

```diff
@@ -23,39 +23,42 @@ Luckily, there are relatively few. This page documents all the terminology
 required to understand and use Packer. The terminology is in alphabetical order
 for quick referencing.
 
-- `Artifacts` are the results of a single build, and are usually a set of IDs
-  or files to represent a machine image. Every builder produces a single
-  artifact. As an example, in the case of the Amazon EC2 builder, the
-  artifact is a set of AMI IDs (one per region). For the VMware builder, the
-  artifact is a directory of files comprising the created virtual machine.
+- `Commands` are sub-commands for the `packer` program that perform some job.
```

Comment:
I think we should try to keep the alphabetical order for this file, it should makes it easier to find things for readers.

Reply:
I didn't realize they were in alphabetical order. I tried keeping them in contextual order. I'll put them back ordered alphabetically.